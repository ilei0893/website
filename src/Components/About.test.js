import About from "./About.js";
import React from "react";
import renderer from "react-test-renderer";

describe("Tests the About section", () => {
  it("matches the snapshot", () => {
    const tree = renderer.create(<About />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
