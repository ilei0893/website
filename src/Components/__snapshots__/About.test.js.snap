// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests the About section matches the snapshot 1`] = `
<section
  id="about"
>
  <div
    className="row"
  >
    <div
      className="three columns"
    >
      <img
        alt="Tim Baker Profile Pic"
        className="profile-pic"
      />
    </div>
    <div
      className="nine columns main-col"
    >
      <h2>
        About Me
      </h2>
      <p />
      <div
        className="row"
      >
        <div
          className="columns contact-details"
        >
          <h2>
            Contact Details
          </h2>
          <p
            className="address"
          >
            <span />
            <br />
            <span />
            <br />
            <span />
          </p>
        </div>
        <div
          className="columns download"
        >
          <p>
            <a
              className="button"
              href="https://github.com/ilei0893/website/blob/master/public/LeiIvanResume.pdf"
            >
              <i
                className="fa fa-clipboard"
              />
              View Resume
            </a>
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
`;
